input/templates/partials/myaccount/my-address-book-adress.hbs

```diff
@@ -0,0 +1,18 @@
+<div>{{address.title}} {{address.firstName}} {{address.lastName}}</div>
+<div>{{address.streetName}}</div>
+<div>{{address.additionalStreetInfo}}</div>
+<div>{{address.region}}</div>
+<div>{{address.postalCode}}</div>
+<div>{{address.country}}</div>
+<div class="address-book-edit-delete">
+  <a href="{{address.editUrl}}">
```

Comment:
I had to use here a value that is coming from the application (instead of `@root.meta._links....`) because that's the only way I can identify which address do I want to modify. In the `editUrl` I will pass an identifier of this address. It's like the URL for a product or category works like.